packages/moonstone/styles/variables.less

```diff
@@ -351,8 +351,8 @@
 
 // ContextualPopup
 // ---------------------------------------
-@moon-contextual-popup-border-width: @moon-button-border-width;
-@moon-contextual-popup-border-radius: 15px;
+@moon-contextual-popup-border-width: 1px;
```

Comment:
`1px` is negligible at higher resolutions (4k and 8k) and isn't currently adjusted by our RI plugin. I think we'll need to resolve this eventually but not necessarily in this PR as it's a broader issue than is being solved here.

Reply:
This is intentional. The border isn't considered as a GUI element, but rather a functional "line" that differentiates between the popup and the outside.